airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/manifest.yaml

```diff
@@ -856,6 +856,97 @@ definitions:
       cursor_filter: "start_time"
       primary_key: "id"
 
+  ticket_metrics_stream:
+    type: StateDelegatingStream
+    name: "ticket_metrics"
+    schema_loader:
+      type: JsonFileSchemaLoader
+      file_path: "./source_zendesk_support/schemas/ticket_metrics.json"
+    full_refresh_stream:
+      $ref: "#/definitions/base_incremental_stream"
+      schema_loader:
+        type: JsonFileSchemaLoader
+        file_path: "./source_zendesk_support/schemas/ticket_metrics.json"
+      name: "ticket_metrics"
+      primary_key: "id"
+      incremental_sync:
+        type: DatetimeBasedCursor
+        cursor_datetime_formats:
+          - "%s"
+          - "%Y-%m-%dT%H:%M:%SZ"
+          - "%Y-%m-%dT%H:%M:%S%z"
+        datetime_format: "%s"
+        cursor_field: "_ab_updated_at"
+        start_datetime:
+          datetime: "0" # not used as the API does not take filters in
+      retriever:
+        $ref: "#/definitions/retriever"
+        ignore_stream_slicer_parameters_on_paginated_requests: true
+        requester:
+          $ref: "#/definitions/retriever/requester"
+          path: "ticket_metrics"
+        paginator:
+          $ref: "#/definitions/links_next_paginator"
+          pagination_strategy:
+            $ref: "#/definitions/after_url_paginator/pagination_strategy"
+            stop_condition: "{{ config.get('ignore_pagination', False) or not response['meta']['has_more'] }}"
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: ["ticket_metrics"]
+      transformations:
+        - type: AddFields
+          fields:
+            - path:
+                - "_ab_updated_at"
+              value: "{{ record['updated_at'] }}"
+    incremental_stream:
+      $ref: "#/definitions/base_incremental_stream"
+      schema_loader:
+        type: JsonFileSchemaLoader
+        file_path: "./source_zendesk_support/schemas/ticket_metrics.json"
+      name: "ticket_metrics"
+      primary_key: "id"
+      incremental_sync:
+        type: DatetimeBasedCursor
+        cursor_datetime_formats:
+          - "%s"
+          - "%Y-%m-%dT%H:%M:%SZ"
+          - "%Y-%m-%dT%H:%M:%S%z"
```

Comment:
`"%Y-%m-%dT%H:%M:%S%z"` this shouldn't break anything, but I was curious why we need to account for this format. looking over `streams.py` I only saw `DATETIME_FORMAT: str = "%Y-%m-%dT%H:%M:%SZ"` and i saw some str -> int conversion methods which accounts for `"%s"

I'm also fine leaving it in since it presumably shouldn't break anything

Reply:
Good catch, removing those!